bootstrapper-maven-plugin/src/main/resources/templates/pom.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>{{groupId}}</groupId>
+    <artifactId>{{artifactId}}</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
+    <name>sample-webpage-operator</name>
+    <packaging>jar</packaging>
+
+    <properties>
+        <java.version>11</java.version>
+        <maven.compiler.source>${java.version}</maven.compiler.source>
+        <maven.compiler.target>${java.version}</maven.compiler.target>
+        <jib-maven-plugin.version>3.4.0</jib-maven-plugin.version>
+        <josdk.version>{{josdkVersion}}</josdk.version>
+        <slf4j.version>1.7.36</slf4j.version>
```

Comment:
Similarly, I don't think that we should add logging stuff here or generate logging configuration.

Reply:
I think it should, the idea is that it works out of the box what is generated, so there are log messages that user can see, but can be changed.